features/support/steps.js

```diff
@@ -36,7 +41,10 @@ Given(/^I set the environment variables to:$/, function (env) {
 });
 
 
-When(/^I run `([^`]+)`$/, function (command) {
+When(/^I run `([^`]+)`$/, function (match) {
+  const command = match.startsWith('dredd ')
+    ? match.replace(/^dredd/, DREDD_BIN)
```

Comment:
When expression doesn't match no replacement will happen. You can simplify this to:

```js
const command = match.replace(/^dredd(?= )/, DREDD_BIN);
```

In case the command starts with `dredd` as has a succeeding space it will be replaced with `DREDD_BIN`. If not, it will be used as-is. Sandbox: https://regex101.com/r/ZjaMH0/1.

Reply:
Thanks! I was grappling with this one, because when it was just `^dredd` it was matching things like `dredd-hooks-python` 😬 I'll use your suggestion.